sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -193,3 +194,38 @@ def test_calinski_harabaz_score():
     labels = [0] * 10 + [1] * 10 + [2] * 10 + [3] * 10
     assert_almost_equal(calinski_harabaz_score(X, labels),
                         45 * (40 - 4) / (5 * (4 - 1)))
+
+
+def test_davies_bouldin_index():
+    rng = np.random.RandomState(seed=0)
+
+    # Assert message when there is only one label
+    assert_raise_message(ValueError, "Number of labels is",
+                         davies_bouldin_index,
+                         rng.rand(10, 2), np.zeros(10))
+
+    # Assert message when all point are in different clusters
+    assert_raise_message(ValueError, "Number of labels is",
+                         davies_bouldin_index,
+                         rng.rand(10, 2), np.arange(10))
+
+    # Assert the value is 0. when all samples are equals
+    assert_equal(0., davies_bouldin_index(np.ones((10, 2)),
```

Comment:
These days we would prefer a bare assert, rather than assert_equal

Reply:
Fixed